site/public/js/ta-grading-rubric.js

```diff
@@ -1769,6 +1769,12 @@ function onRestoreMark(me) {
  * @param me DOM Element of the delete button
  */
 function onDeleteComponent(me) {
+    const componentCount = $('.component-container').length;
+    if (componentCount === 1) {
+        alert('Cannot delete the only component.');
```

Comment:
Would it be better to use displayErrorMessage() to have the red banner message instead of the JS alert?

Reply:
Ye I have switched to displayErrorMessage instead now
![image](https://github.com/Submitty/Submitty/assets/122962727/5d9db405-a420-4e38-851d-e7c21da51a76)